src/main/java/agh/ics/oop/OptionsParser.java

```diff
@@ -0,0 +1,21 @@
+package agh.ics.oop;
+
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class OptionsParser {
+    public static List<MoveDirection> parse(String[] args) {
+        List<MoveDirection> directions = new ArrayList<MoveDirection>();
+        MoveDirection[] availableDirections = {MoveDirection.FORWARD, MoveDirection.BACKWARD, MoveDirection.RIGHT, MoveDirection.LEFT};
+        String[] correctDirections = {"f", "b", "r", "l", "forward", "backward", "right", "left"};
+        for (String element : args) {
+            int index = Arrays.asList(correctDirections).indexOf(element);

```

Comment:
całkiem fajny pomysł, choć tu wydajnościowo lepiej byłoby tę listę stworzyć raz, przed pętlą... a najlepiej to w ogóle zamiast listy użyć tutaj mapę `String` ->`MoveDirection`. ;) O kolekcjach jeszcze będziemy mówić, grunt że jakoś sobie tu Pan poradził.